src/analog/commands/downlink/GetExAbsCurrentMC.ts

```diff
@@ -24,7 +24,7 @@ const examples: TCommandExampleList = [
  *
  * // output command binary in hex representation
  * console.log(command.toHex());
- * // 1f 0f 00
+ * // 1f 0f 0a
```

Comment:
is it correct?

Reply:
there is no `0a` in the example of the command. 
`
const examples: TCommandExampleList = [
    {
        name: 'simple request',
        hex: {header: '1f 0f 00', body: ''}
    }
];
`

Reply:
fixed